roles/dotfiles/tasks/main.yml

```diff
@@ -12,6 +11,12 @@
     dest: ~/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
     force: yes
 
+- name: Clone zsh-autosuggestions
+  git:
+    repo: https://github.com/zsh-users/zsh-autosuggestions
+    dest: ~/.oh-my-zsh/custom/plugins/zsh-syntax-autosuggestions
```

Comment:
Plugin is not found.

```suggestion
    dest: ~/.oh-my-zsh/custom/plugins/zsh-autosuggestions
```

Reply:
![clean](https://user-images.githubusercontent.com/523981/68383232-90b74800-0155-11ea-876b-5823598c4080.gif)